src/game/start.py

```diff
@@ -11,14 +12,42 @@ def start_game():
     pieces_list = Piece.create_pieces_list()
 
     game_state = GameState(parameter)
-    if len(game_state.message) > 0:
-        error_message = game_state.message
-
     if len(error_message) > 0:
```

Comment:
You can do this in one shot like it:

if len(error_message) > 0:
    game_state.message = error_message += """\nValid sample : --state='{"grid" : {"A2": 10,"C1":3,"D1":12},"turn" :{"player" : 1,"selected" : 7}}'"""